lightning/src/ln/channel.rs

```diff
@@ -3854,6 +3875,9 @@ impl<Signer: Sign> Channel<Signer> {
 		if let Some((feerate, _)) = self.pending_update_fee {
 			feerate_per_kw = cmp::max(feerate_per_kw, feerate);
 		}
+		if let Some(feerate) = outbound_feerate_update {
```

Comment:
Hmm, I wonder if we should be conservative here and just always use *any* pending fee if there's a pending fee. That way even if our counterparty bumps the fee and increases the dust buffer we'll stop accepting HTLCs before they even commit the fee.

Reply:
Do you mean if `outbound_feerate_update.is_some()` and `self.pending_update_fee.is_some()` only compute with the latter ? I think it's not possible as you can't have both parties being the fundee and as such the fee responsibility...but likely you mean something else ?

Reply:
Oh, I think you're right, I'm not sure what point I was trying to make here.